test-record-interceptor/src/main/java/dev/cgomez/testrecordinterceptor/FileRecorded.kt

```diff
@@ -0,0 +1,35 @@
+package dev.cgomez.testrecordinterceptor
+
+import okhttp3.Protocol
+import okhttp3.Request
+import okhttp3.RequestBody.Companion.toRequestBody
+import okhttp3.Response
+import okhttp3.ResponseBody.Companion.toResponseBody
+
+data class FileRecorded(
+  val responseRecorded: FileResponse,
```

Comment:
I like this naming convention, can we use it for the typings too?
maybe something like `RecordedResponse` & `RecordedRequest` is more accurate.

Reply:
absolutely! I changed the file names to be more accurate.